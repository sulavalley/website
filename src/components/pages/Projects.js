import React from 'react';
import base from 'base';
import map from 'lodash/map';
import Project from './Project';

class Projects extends React.Component {
  getProjects() {
    base.fetch('projects', {
      context: this,
      asArray: true,
      then(data) {
        console.log(data);
      }
    });
  }

  render() {
    const { projects } = this.getProjects;

    return <Project />;
  }
}

export default Projects;
