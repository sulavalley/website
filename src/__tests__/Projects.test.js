import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Projects from './../components/pages/Projects';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Projects />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders without crashing', () => {
  shallow(<Projects />);
});

test('renders correctly', () => {
  const tree = renderer.create(<Projects />).toJSON();
  expect(tree).toMatchSnapshot();
});
