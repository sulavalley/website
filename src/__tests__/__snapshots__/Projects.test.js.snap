// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="ui four column grid"
  />
</div>
`;
